core/src/main/java/com/adobe/aem/commons/assetshare/content/renditions/impl/dispatchers/InternalRedirectRenditionDispatcherImpl.java

```diff
@@ -138,6 +143,57 @@ public void dispatch(SlingHttpServletRequest request, SlingHttpServletResponse r
         }
     }
 
+    @Override
+    public AssetRendition getRendition(final AssetModel assetModel, final AssetRenditionParameters parameters) {
+        throw new UnsupportedOperationException(String.format("[ %s ] is not supported by the AEM Async Asset Download Framework.",
+                this.getClass().getName()));
+
+        /*
+        // Commented out implementation for AEM as a Cloud Service as this only supports a sub-set of the original use-case.
```

Comment:
@davidjgonzalez should this get removed? or did you want to leave it in here...

Reply:
Ugh - Felt bad throwing it away since it's implemented. Guess we can remove it tho... gonna lose it to the squash